src/save_index.cpp

```diff
@@ -405,9 +405,11 @@ void extract_summary_from_config(config& cfg_save, config& cfg_summary)
 				color_options_ = game_config::default_colors;
 				color_id_ = color_options_[color_];
 				
-				if(u["color"].to_int()){
-				color_ = u["color"].to_int() - 1;
-				color_id_ = color_options_[color_];
+				if (!cfg[ "color" ]. empty ()) {
+					if(u["color"].to_int()){
```

Comment:
Instead of two if statements, just use a single one with `if(cfg.has_attribute("color")`.